x-pack/plugins/index_lifecycle_management/public/application/services/navigation.ts

```diff
@@ -4,6 +4,20 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const getPolicyPath = (policyName: string): string => {
+export const ROUTES = {
```

Comment:
Calling this const `REACT_ROUTER_ROUTES` could make this even clearer. Since the strings are coupled to how react router works.

Reply:
I'd prefer keeping ROUTES variable name here for shortness sake and it seemed to me as a good convention to use for a react-router config paths.